src/services/assetService.ts

```diff
@@ -182,7 +182,7 @@ export class AssetService {
             if (nodejsMode) {
                 const FileType = require('file-type');
                 const fileType = await FileType.fromFile(normalizedPath);
-                types = [fileType.ext];
+                types = fileType?.ext;
```

Comment:
variable "types" should still be an array. 
I think `types = fileType?.ext ? [fileType.ext] : []` is what you want